week3/homework/src/actions/readTodo.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const fs = require('fs');
+const path = '../todos.json';
+
+function readTodo(req, res) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(path, 'utf8', (error, todos) => {
+      if (error) {
+        reject(error);
+      }
+      else {
+        const parsedTodos = JSON.parse(todos);
+        const todo = parsedTodos.filter(todo => todo.id === req.params.id);
```

Comment:
I am curious. Why do you sometimes use `filter `and other times `find `?

Reply:
No particular reason! Do u think is it best practice to use one than the other? 
Thanks in advance for your feedback!

Reply:
Not so much best practice, but being consistent is always preferred. If you are looking for exactly one element I think find is more clear than filter.